src/ceph_mon.cc

```diff
@@ -156,7 +156,7 @@ int check_mon_data_empty()
   return code;
 }
 
-void usage()
+void print_mon_usage()
```

Comment:
can we just make this function static? or hide it in an anonymous namespace? the same applies to changes to other `usage()` functions.